gradle/dependencies.gradle

```diff
@@ -4,18 +4,14 @@ repositories {
     mavenCentral()
 
     maven {
-        url = 'https://oss.jfrog.org/artifactory/oss-snapshot-local'
-    }
-
-    maven {
-        url = 'https://dl.bintray.com/corfudb/maven'
+        url = 'https://corfudbcloud.jfrog.io/artifactory/corfu-cloud/'
     }
 }
 
-configurations.all {
-    resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
-    resolutionStrategy.cacheDynamicVersionsFor 0, 'seconds'
-}
+//configurations.all {
```

Comment:
This piece of configuration is very critical for automation. It checks if there is a new version of an artifact every single time. Since we use SNAPSHOTS, it's critical to have that functionality. 
Could you please uncomment

Reply:
Done, thanks!